http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -241,9 +241,6 @@ class ProxyRoutesSpec
       status shouldEqual StatusCodes.OK
       validateCors()
     }
-    Get(s"/api/google/v1/apps/foo").addHeader(Cookie(tokenCookie)) ~> httpRoutes.route ~> check {
```

Comment:
not too sure why this was in the proxy route spec, but it succeeds now that we've changed app -> apps. Seems  out of place in the context of the rest of the test code.

Reply:
Think it was trying to assert that the proxy routes don't listen on `/api`… but yeah, fine to remove it.